__tests__/buildTranslationFiles.spec.ts

```diff
@@ -109,8 +130,8 @@ describe('buildTranslationFiles', () => {
           expected[nonNumericKey] = defaultValue;
         });
 
-        buildTranslationFiles(config);
-        assertTranslation({ type, expected });
+        buildTranslationFiles({ ...config, outputFormat });
```

Comment:
This seems very repetitive, maybe we should wrap it with a function that expects the config?
Something like: 
```ts
createTranslations(config)

function createTranslations(config) {
 buildTranslationFiles({ ...config, outputFormat });
}
```
What do you think?